### PR TITLE
[homework#20]-react-api by Vitalina Poremchuk

### PR DESCRIPTION
**Pull request by PoremchukV**

### CODE REVIEW

File: homeworks/vitalina-poremchuk/20-api/my-app/src/Form.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
```
Comment (esvyridov) [inline homeworks/vitalina-poremchuk/20-api/my-app/src/Form.js:1]: file is not used

File: homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            <p>{message}</p>
+            <Button onClick={handlerModal}>Close</Button>
+          </Modal>
+        )}
+        {people.map((person, index) => (
+          <Image
+            key={index.toString()}
+            onClick={() => refreshAvatar(index)}
+            src={person.avatars[0].url}
+            loader={loader ? loader.toString() : null}
```
Comment (esvyridov) [inline homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx:82]: ternary operator and `.toString` can be omitted

File: homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      <div className="container__card">
+        {modal && (
+          <Modal onClick={handlerModal}>
+            <p>{message}</p>
+            <Button onClick={handlerModal}>Close</Button>
+          </Modal>
+        )}
+        {people.map((person, index) => (
+          <Image
+            key={index.toString()}
```
Comment (esvyridov) [inline homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx:79]: `.toString` can be omitted

File: homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    setLoader(true);
+    const refreshAvatars = await Promise.all(
+      people.map((avatar) => getAvatar()),
+    );
+    setPeople(refreshAvatars);
+    setLoader(false);
+  }
+
+  function handlerModal() {
```
Comment (esvyridov) [inline homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx:64]: can we rename this function to make it obvious what is happening inside?

**Review by esvyridov — APPROVED** [review]

File: homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx (hunk 2 above)
Comment (PoremchukV) [reply homeworks/vitalina-poremchuk/20-api/my-app/src/container/Fetch/Fetch.jsx:82]: I can't pass a boolean to `className`.

**Pull request merged**